The 20.4.24 release has a default cross_config file that is readily
compatible with the TS-7820/40 in regards to the utils package. This
just patches the file in so it can be used.

--- /dev/null	2020-07-27 08:52:25.842867328 -0700
+++ b/src/util/android-arm.cross_config	2020-07-27 16:46:15.684739786 -0700
@@ -0,0 +1,21 @@
+CL_FIXNUM_TYPE=int
+CL_FIXNUM_BITS=32
+CL_FIXNUM_MAX=536870911L
+CL_FIXNUM_MIN=-536870912L
+CL_INT_BITS=32
+CL_LONG_BITS=32
+ECL_STACK_DIR=down
+ECL_BIGENDIAN=no
+ECL_NEWLINE=LF
+ECL_FILE_CNT=0
+ECL_STDINT_HEADER="#include <stdint.h>"
+ECL_UINT8_T=uint8_t
+ECL_UINT16_T=uint16_t
+ECL_UINT32_T=uint32_t
+ECL_UINT64_T=uint64_t
+ECL_INT8_T=int8_t
+ECL_INT16_T=int16_t
+ECL_INT32_T=int32_t
+ECL_INT64_T=int64_t
+ECL_LONG_LONG_BITS=64
+ECL_WORKING_ENVIRON=yes
